policyengine_us/parameters/gov/states/ok/tax/income/rates/head_of_household.yaml

```
@@ -27,22 +27,27 @@ brackets:
       2022-01-01: 0.0375
   - threshold:
       2021-01-01: 12_200
+      2024-01-01: 14_400
```

While you're here could you leave comments tying this to the statutes? For example (C)(2)(a) says:
```suggestion
      #  68 O.S. § 2355(C)(2)(a) defines the bracket structure incrementally.
      # $2,000 + $3,000 + $2,500 + $2,300 + $4,600 = $14,400
      2024-01-01: 14_400
```
Here and elsewhere.